src/components/TheNav.vue

```diff
@@ -146,7 +77,57 @@ export default {
   name: 'TheNav',
   data() {
     return {
-      isNavShown: false
+      isNavShown: false,
+      navItems: [
+        {
+          path: { name: 'home' },
+          activeCondition: 'home',
+          content: 'Home',
+          conditionOne: true,
+          conditionTwo: false,
+          function: ''
+        },
+        {
+          path: { name: 'create' },
+          activeCondition: 'create',
+          content: 'Create new workout',
+          conditionOne: true,
+          conditionTwo: false,
+          function: ''
+        },
+        {
+          path: { name: 'dashboard' },
+          activeCondition: 'dashboard',
+          content: 'Dashboard',
+          conditionOne: true,
+          conditionTwo: false,
+          function: ''
+        },
+        {
+          path: { name: 'sign-in' },
+          activeCondition: 'sign-in',
+          content: 'Sign in',
+          conditionOne: false,
+          conditionTwo: false,
+          function: ''
+        },
+        {
+          path: { name: 'sign-up' },
+          activeCondition: 'sign-up',
+          content: 'Sign up',
+          conditionOne: false,
+          conditionTwo: false,
+          function: ''
+        },
+        {
+          path: { name: 'home' },
+          activeCondition: false,
+          content: 'Sign out',
+          conditionOne: true,
+          conditionTwo: true,
+          function: this.signOut()
+        }
```

Comment:
Don't use `function` keyword, this is a reserved word for the language interpreter. It should be a `callback`

Reply:
**Corrected**